composer.json

```diff
@@ -11,7 +11,7 @@
         "php": "^7.4 || ^8.0",
         "ext-json": "*",
         "league/oauth1-client": "^1.9",
-        "socialiteproviders/manager": "~4.0"
+        "socialiteproviders/manager": "~4.2"
```

Comment:
```suggestion
        "socialiteproviders/manager": "~4.0"
```

Reply:
As an FYI, @atymic asked me to update this to `^4.2`... see his comments above. But I have downgraded as per your request.